### PR TITLE
RCAL-299 Create RefFileName Schema

### PR DESCRIPTION
**Pull request by cshanahan1**

### CODE REVIEW

File: src/rad/resources/schemas/ref_file-1.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            title: Linearity file name
+            type: string
+      persat:
+        title: PERSAT reference file information
+        type: object
+        properties:
+          name:
+            title: PERSAT file name
+            type: string
+      trapdensity:
```
[the remark this answers is not in the document]
Comment (nden) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:70]: Look at the reference file types here https://roman-crds-test.stsci.edu/ and remove those that are not in the list.

[the remark this answers is not in the document]
Comment (cshanahan1) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:70]: I was referencing this page: https://innerspace.stsci.edu/display/SCSB/Roman+Reference+Files

i should just do the smaller set of files in crds?

[the remark this answers is not in the document]
Comment (cshanahan1) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml]: i just changed this to the smaller set.

[the remark this answers is not in the document]
Comment (nden) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml]: Yes, because we may not need some of these for Roman and we'll add them as needed.

**Review by nden — APPROVED** [review]
LGTM

File: src/rad/resources/schemas/ref_file-1.0.0.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            title: Linearity file name
+            type: string
+      photom:
+        title: Photometry reference file information
+        type: object
+        properties:
+          name:
+            title: Photometry file name
+            type: string
+            blend_table: True
```
Comment (PaulHuwe) [inline src/rad/resources/schemas/ref_file-1.0.0.yaml:70]:
```suggestion
```

We aren't using blend tables.

**Review by PaulHuwe — APPROVED** [review]
Outside of the extraneous blend_table line, LGTM.

**Review by gardner358 — COMMENTED** [review]
Are any of these fields going to be in the archive catalog?

File: src/rad/resources/schemas/ref_file-1.0.0.yaml (hunk 2 above)
Comment (nden) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:70]: @cshanahan1 Can you remove the `blend_table` attribute so we can merge this?

**Comment by nden** [comment]
@gardner358 : As per @ddavis-stsci , none of this needs to go in the archive catalog.

**Pull request merged**